modules/api/src/main/java/org/eclipse/xpanse/api/config/GetCspMethodInfo.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.api.config;
+
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Get custom annotations for csp information.
+ */
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface GetCspMethodInfo {
+
+    /**
+     * The class of the method can get csp info.
+     */
+    Class<?> clazz() default GetCspMethodHelper.class;
+
+    /**
+     * The method name can get csp info.
+     */
+    String methodName() default "";
+
+
+    /**
+     * The param types of the method can get csp info.
+     */
+    Class<?>[] paramTypes() default {String.class};
+
+    /**
+     * The index of params can be used to get csp info with the method.
+     */
+    int[] paramIndexes() default {0};
+
+    /**
+     * Identifies whether csp can be obtained directly.
+     */
+    boolean enabled() default true;
```

Comment:
two booleans fields. It is confusing. 
My understanding is that the same annotation can be used for all types of methods. So the only boolean flag is necessary to disable audit completely.